### PR TITLE
updated cpc docs

### PR DESCRIPTION
**Pull request by ananyahjha93**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?   
- [ ] Did you write any new necessary tests?  

## What does this PR do?
Fixes # (issue).

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pl_bolts/models/self_supervised/cpc/cpc_module.py [hunk 1]
```diff
@@ -49,59 +49,6 @@ def __init__(
             **kwargs,
     ):
         """
-        PyTorch Lightning implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding
-        <https://arxiv.org/abs/1905.09272>`_
-
-        Paper authors: (Olivier J. Hénaff, Aravind Srinivas, Jeffrey De Fauw, Ali Razavi,
-        Carl Doersch, S. M. Ali Eslami, Aaron van den Oord).
```
Comment (Borda) [inline pl_bolts/models/self_supervised/cpc/cpc_module.py:56]: lets keep these lines...

Comment (ananyahjha93) [reply pl_bolts/models/self_supervised/cpc/cpc_module.py:56]: this has been moved to the rst file

File: pl_bolts/models/self_supervised/cpc/cpc_module.py [hunk 2]
```diff
@@ -49,59 +49,6 @@ def __init__(
             **kwargs,
     ):
         """
```
Comment (Borda) [inline pl_bolts/models/self_supervised/cpc/cpc_module.py:51]:
```suggestion
        """
        PyTorch Lightning implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding	
        <https://arxiv.org/abs/1905.09272>`_

        Paper authors: (Olivier J. Hénaff, Aravind Srinivas, Jeffrey De Fauw, Ali Razavi,	
        Carl Doersch, S. M. Ali Eslami, Aaron van den Oord).
```

File: pl_bolts/models/self_supervised/cpc/cpc_module.py [hunk 3]
```diff
@@ -49,59 +49,11 @@ def __init__(
...
-        PyTorch Lightning implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding
+        PyTorch Lightning implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding
```
Comment (Borda) [inline pl_bolts/models/self_supervised/cpc/cpc_module.py:52]:
```suggestion
        PyTorch Lightning implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding
```

File: pl_bolts/models/self_supervised/cpc/cpc_module.py [hunk 4]
```diff
@@ -49,59 +49,11 @@ def __init__(
             **kwargs,
     ):
         """
-        PyTorch Lightning implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding
+        PyTorch Lightning implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding	
         <https://arxiv.org/abs/1905.09272>`_
 
-        Paper authors: (Olivier J. Hénaff, Aravind Srinivas, Jeffrey De Fauw, Ali Razavi,
+        Paper authors: (Olivier J. Hénaff, Aravind Srinivas, Jeffrey De Fauw, Ali Razavi,
```
Comment (Borda) [inline pl_bolts/models/self_supervised/cpc/cpc_module.py:55]:
```suggestion
        Paper authors: (Olivier J. Hénaff, Aravind Srinivas, Jeffrey De Fauw, Ali Razavi,
```

**Comment by Borda** [comment]
@ananyahjha93 is it ready?

**Review by Borda — APPROVED** [review]

**Pull request merged**